webrisk/webrisk_client.go

```diff
@@ -112,6 +112,7 @@ func (tt ThreatType) String() string { return pb.ThreatType(tt).String() }
 
 // List of ThreatType constants.
 const (
+	ThreatType_Unspecified			 = ThreatType(pb.ThreatType_THREAT_TYPE_UNSPECIFIED)
```

Comment:
Noticing this now, generated code is not held to the same standard, but if this is wrapping that, it can use normal Go naming conventions (camelCase, not underscores).

https://golang.org/doc/effective_go.html#mixed-caps

Reply:
Done.